static/js/fit.js

```diff
@@ -731,6 +775,114 @@ $( document ).ready(function() {
+            window.location.href = `/course/${JSON.parse(xhr.response)['slug']}/edit`
+        } else {
+                console.log('DEV: give me some message')
+            }
+    });
+  });
+
+  delegate('a[data-course-edit-remove-teacher]', 'click', (e, t) => {
+    let teacherId = e.target.dataset.teacherId;
+    let slug = t.dataset.courseSlug;
+    post(`/course/${slug}/edit/remove/teacher/${teacherId}`, {}, (responseText, xhr) => {
+        let responseJSON = JSON.parse(xhr.response);
+        let alert = $('#add-teacher-alert');
+        if (xhr.status == 200) {
+            let teacherDiv = $(`#teacher-${responseJSON['teacher_id']}`);
+            if (teacherDiv) {
+                teacherDiv.remove()
+            }
+        } else {
+                alert.css("display", "block");
+                alert.removeClass('alert-success');
+                alert.addClass('alert-danger');
+                alert.html(responseJSON.message);
+            }
+    });
+  });
+
+  delegate('#add-teacher', 'click', (e, t) => {
+    e.preventDefault();
+    let email = document.querySelector('#add-teacher-email');
+    let teacherId = e.target.dataset.teacherId;
+    let slug = t.dataset.courseSlug;
+    post(`/course/${slug}/edit/add/teacher`, {teacher_email: email.value}, (responseText, xhr) => {
+        let alert = $('#add-teacher-alert');
+        let responseJSON = JSON.parse(xhr.response);
+        alert.css("display", "block");
+        if (xhr.status == 400) {
+            alert.removeClass('alert-success');
+            alert.addClass('alert-danger');
+        } else {
+            alert.addClass('alert-success');
+            alert.removeClass('alert-danger');
+            $('#teachers-list').append(
+                `<div id="teacher-${responseJSON['teacher']['id']}" class="fit_btn">
+                      <div class="fit_pic circle"><img src="${responseJSON['teacher']['picture']}" alt="user"></div>
+                      <div class="fit_txt">${responseJSON['teacher']['first_name']} ${responseJSON['teacher']['last_name']}</div>
+                      <div class="sub buttonset">
+                        <a class="btn btn-sm btn-light" data-teacher-id="${responseJSON['teacher']['id']}" data-course-slug="${responseJSON['teacher']['slug']}" data-course-edit-remove-teacher><i class="fal fa-trash"></i> Remove</a>
+                      </div>
+                    </div>`
+
+            )
+        }
+        alert.html(responseJSON.message);
+    });
+  });
+
+  $('#fit_modal_delete').on('show.bs.modal', function(event){
+    document.querySelector('#confirm-delete').href = event.relatedTarget.href;
+  });
+
+  delegate('#confirm-delete', 'click', (e, t) => {
+    e.preventDefault();
+    post(t.href, {}, (responseText, xhr) => {
+        if (xhr.status == 200) {
+            window.location.href = JSON.parse(xhr.response)['success_url']
+        } else {
+                console.log(JSON.parse(xhr.response)['message'])
```

Comment:
`console.log` statements are debug and should be removed before committing.

You can see requests coming through the site using the Requests tab in the Inspector.